src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -119,6 +146,30 @@ export class BrowsingContextProcessor {
       );
     });
 
+    sessionCdpClient.Page.on(
+      'lifecycleEvent',
```

Comment:
In Puppeteer, it's done on `frameNavigated`

Reply:
Thanks! Changed to `Page.frameNavigated` as well.